.gitlab/kitchen_testing/new-e2e_testing/ubuntu.yml

```diff
@@ -21,31 +21,37 @@
     E2E_OSVERS: "ubuntu-14-04,ubuntu-16-04,ubuntu-18-04,ubuntu-20-04,ubuntu-22-04"
     E2E_CWS_SUPPORTED_OSVERS: "ubuntu-18-04,ubuntu-20-04,ubuntu-22-04"
     E2E_BRANCH_OSVERS: "ubuntu-22-04"
-  needs: ["deploy_deb_testing-a6_x64"]
+  needs:
+    - !reference [ .needs_new_e2e_template ]
+    - deploy_deb_testing-a6_x64
 
 .new-e2e_ubuntu_a6_arm64:
   variables:
     E2E_ARCH: arm64
     E2E_OSVERS: "ubuntu-18-04,ubuntu-20-04"
     E2E_CWS_SUPPORTED_OSVERS: "ubuntu-18-04,ubuntu-20-04"
     E2E_BRANCH_OSVERS: "ubuntu-20-04"
-  needs: ["deploy_deb_testing-a6_arm64"]
+  needs:
+    - !reference [ .needs_new_e2e_template ]
+    - deploy_deb_testing-a6_arm64
 
 .new-e2e_ubuntu_a7_x86_64:
   variables:
     E2E_ARCH: x86_64
     E2E_OSVERS: "ubuntu-14-04,ubuntu-16-04,ubuntu-18-04,ubuntu-20-04,ubuntu-22-04"
     E2E_CWS_SUPPORTED_OSVERS: "ubuntu-18-04,ubuntu-20-04,ubuntu-22-04"
     E2E_BRANCH_OSVERS: "ubuntu-22-04"
-  needs: ["deploy_deb_testing-a7_x64"]
+  needs:
+    - !reference [ .needs_new_e2e_template ]
+    - deploy_deb_testing-a7_x64
 
 .new-e2e_ubuntu_a7_arm64:
   variables:
     E2E_ARCH: arm64
     E2E_OSVERS: "ubuntu-18-04,ubuntu-20-04"
     E2E_CWS_SUPPORTED_OSVERS: "ubuntu-18-04,ubuntu-20-04"
     E2E_BRANCH_OSVERS: "ubuntu-20-04"
-  needs: ["deploy_deb_testing-a7_arm64"]
+  needs: ["deploy_deb_testing-a7_arm64", "go_e2e_deps"]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should this be the `.needs_new_e2e_template`?

Reply:
good catch!